dwa_local_planner:
  ros__parameters:
    max_vel_x:        3.0
    min_vel_x:        0.0
    max_yaw_rate:     2.5
    acc_x:            3.0
    acc_yaw:          3.0
    v_res:            0.3
    w_res:            0.3
    dt:               0.1
    predict_time:     2.0

    weight_heading:   1.0
    weight_path:      1.0
    weight_obstacle:  1.0
    weight_velocity:  0.5

    robot_radius:     0.3
    safety_distance:  0.3
    wheel_base:       0.33
    look_ahead_dist:  1.0

